source/meetings/general/2023-01-24.rst

```diff
@@ -0,0 +1,43 @@
+General Meeting 09:00 18-Sep-2023
+=================================
+
+Held at City Hall, Cardiff during PyCon UK 2023 and chaired by Owen Campbell.
+
+Trustees Present
+----------------
+
+Kristian Glass, Becky Smith, Owen Campbell, Vince Knight, Ann Barr
+
+
+Report
+------
+
+Owen Campbell presented the report to the members. Owen presented the
+achievements for the year:
+
+- PyCon UK ran again in 2022 after being postponed during the pandemic.
+  This is a notable achievement given the difficulties related to the pandemic.
+- The UKPA owns UKPA Trading Limited and for the first time the company was
+  ready for the conference. There is still some more progress to be made:
+  surplus funds from another entity need to be moved.
+- The UKPA supported the running of DjangoCon Europe. This is the first event
```

Comment:
```suggestion
- The UKPA provided business and operational support for the running of DjangoCon Europe. This is the first event
```

nitpick: I'd like to be a bit more specific about "support" if that's ok - we can support through providing money/advice, which are certainly valuable, but we supported through providing a backing entity, which is not only valuable, but a heck of a lot more work